packages/core/src/components/pagination/stories/pagination.react.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { AtomPagination } from '@juntossomosmais/atomium/react'
+import { Meta, StoryObj } from '@storybook/react'
+import React from 'react'
+
+import { PaginationComponentArgs, PaginationStoryArgs } from './pagination.args'
+
+export default {
+  title: 'Components/Pagination',
+  component: AtomPagination,
+  ...PaginationStoryArgs,
+} as Meta
+
+const createPagination = (args: typeof PaginationComponentArgs) => (
+  <AtomPagination count={args.count} page={args.page} />
+)
+
+export const Default: StoryObj<typeof PaginationComponentArgs> = {
```

Comment:
suggestion: add a new example with fewer pages (ex: 3 pages)

Reply:
@tassioFront I didn't feel much need to add more examples because in the control can already reproduce most of the examples in a simple way.

<img width="1084" alt="Captura de Tela 2024-12-06 às 11 09 42" src="https://github.com/user-attachments/assets/451aadd6-5bb3-4924-a543-685f86fa2ae5">

Reply:
that makes all sense :rocket: